mmedit/models/backbones/encoder_decoders/aot_encoder_decoder.py

```diff
@@ -0,0 +1,68 @@
+import torch.nn as nn
+from mmcv.runner import auto_fp16, load_checkpoint
+
+from mmedit.models.builder import build_component
+from mmedit.models.registry import BACKBONES
+from mmedit.utils import get_root_logger
+
+
+@BACKBONES.register_module()
+class AOTEncoderDecoder(nn.Module):
```

Comment:
It seems that this class is identical to `GLEncoderDecoder`. You can inherit from `GLEncoderDecoder` by using
```
class AOTEncoderDecoder(GLEncoderDecoder):
```

In this case, you do not need to rewrite the function here. But please make sure the docstring is modified.